_docs/style/text-conventions/date-time.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Date and time
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Rackspace is a global company, so it is critical to ensure that we represent time as clearly and consistently as possible to our customers.
+resource: false
+status: complete
+last-modified: 2018-06-18
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
```

Comment:
@palindromebart, quick note that I have completely refactored this article, pulling in material from the Rackspace style guide, and doing away with this big, bulky tables that were visually overwhelming to me. Please review...as always, this content is up for debate - but hopefully, we're closer!